site/content/user-guide/amqp-adapter.md

```diff
@@ -95,9 +120,21 @@ Note how verbose the address is for unauthenticated devices. This address can be
 
 Publish some JSON data for device `4711`:
 
-    $ java -jar AmqpSend.jar -b "localhost:4040" -to "telemetry/DEFAULT_TENANT/4711" -m '{"temp": 5}'
+    $ /hono/cli/target$ java -jar hono-cli-0.7-SNAPSHOT-exec.jar --spring.profiles.active=amqp-adapter-cli --username="" --password=""
```

Comment:
are you sure this works? IIRC the command line client only checks the username and password fields to not be `null` ...

Reply:
yes, thanks for spotting this.. am reading your comments just now (after already having created a PR to fix this for the client). i made the changes locally but included it in a different patch